plugins/org.eclipse.fordiac.ide.model.search/src/org/eclipse/fordiac/ide/model/search/SearchNameDictionary.java

```diff
@@ -80,6 +82,9 @@ public String hierarchicalName(final Object element) {
 				&& call.eContainer() instanceof final INamedElement namedElement) {
 			return namedElement.getName() + "." + call.getName(); //$NON-NLS-1$
 		}
+		if (element instanceof final LibraryElement libElement) {
+			return PackageNameHelper.getFullTypeName(libElement);
```

Comment:
I think the cleaner solution here is to use  libElement.getTypeEntry().getFullTypeName